ATM.py

```diff
@@ -30,31 +35,59 @@ def select(self):
                             'To withdraw money press 3\n'
                             'To change password press 4\n'
                             'for Exit press -1\n')
-        return self.choice
+        if self.choice == BALANCE_CHOICE:
```

Comment:
make it a function and use it in all place you print the user balance

Reply:
done